cdhweb/blog/models.py

```diff
@@ -37,6 +39,29 @@ def author_list(self):
         return ', '.join(str(user) for user in self.users.all())
     author_list.short_description = 'Authors'
 
+class BlogPostQuerySet(models.QuerySet):
+
+    def featured(self):
+        '''return blog posts that are marked as featured'''
+        return self.filter(is_featured=True)
+
+    def recent(self):
+        '''sort blog posts by publication date'''
+        return self.order_by('-publish_date')
+
+class BlogPostManager(DisplayableManager):
+
+    def get_queryset(self):
+        '''Return default queryset :class:`BlogPostQuerySet`'''
+        return BlogPostQuerySet(self.model, using=self._db)
+
+    def featured(self):
```

Comment:
This seems like exactly what we want to avoid duplication. This happens with other models too, so maybe I can make a separate issue dedicated to refactoring for it?

Reply:
@thatbudakguy great idea - please go ahead and create an issue to document this so we can prioritize it for later work.